source/features/faster-pr-diff-options.tsx

```diff
@@ -43,44 +53,27 @@ function createWhitespaceButton(): HTMLElement {
 		url.searchParams.set('w', '1');
 	}
 
+	const classes = pageDetect.isPR() ?
+		'tooltipped tooltipped-s d-none d-lg-block color-icon-secondary ' + (isHidingWhitespace ? '' : 'color-icon-info') :
+		'tooltipped tooltipped-s btn btn-sm btn-outline tooltipped ' + (isHidingWhitespace ? 'bg-gray-light text-gray-light color-text-tertiary' : '');
+
 	return (
 		<a
 			href={url.href}
 			data-hotkey="d w"
-			className={`btn btn-sm btn-outline tooltipped tooltipped-s ${isHidingWhitespace ? 'bg-gray-light text-gray-light color-text-tertiary' : ''}`}
-			aria-label={`${isHidingWhitespace ? 'Show' : 'Hide'} whitespace in diffs`}
+			className={classes}
+			aria-label={`${isHidingWhitespace ? 'Show' : 'Hide'} whitespace changes`}
 		>
-			{isHidingWhitespace && <CheckIcon/>} No Whitespace
+			{pageDetect.isPR() ? <DiffModifiedIcon/> : <>{isHidingWhitespace && <CheckIcon/>} No Whitespace</>}
 		</a>
 	);
 }
 
-function wrap(...elements: Node[]): DocumentFragment {
-	if (pageDetect.isSingleCommit() || pageDetect.isCompare()) {
-		return (
-			<div className="float-right">
-				{elements.map(element => <div className="ml-3 BtnGroup">{element}</div>)}
-			</div>
-		);
-	}
-
-	return <>{elements.map(element => <div className="diffbar-item">{element}</div>)}</>;
-}
-
-function init(): false | void {
-	const container = select([
-		'#toc', // In single commit view
-		'.pr-review-tools' // In review view
-	]);
-	if (!container) {
-		return false;
-	}
-
-	container.prepend(
-		wrap(
-			createDiffStyleToggle(),
-			createWhitespaceButton()
-		)
+function initPR(): false | void {
+	const container = select('.js-file-filter')!.closest('.flex-auto')!;
+	container.append(
```

Comment:
Variable not necessary
```suggestion
	select('.js-file-filter')!.closest('.flex-auto')!.append(
```